src/ios/CDVSystemBrowser.m

```diff
@@ -0,0 +1,48 @@
+#import "CDVSystemBrowser.h"
+#import <Cordova/CDVPluginResult.h>
+
+@implementation CDVSystemBrowser
+
+const int INITIAL_STATUS_BAR_STYLE = -1;
+
+
+-(void) openUrl:(NSString*)url {
+        CDVPluginResult* pluginResult;
+
+    if (url != nil) {
+    #ifdef __CORDOVA_4_0_0
+        NSURL* baseUrl = [self.webViewEngine URL];
+    #else
+        NSURL* baseUrl = [self.webView.request URL];
+    #endif
+        NSURL* absoluteUrl = [[NSURL URLWithString:url relativeToURL:baseUrl] absoluteURL];
+
+        if ([[UIApplication sharedApplication] canOpenURL:absoluteUrl]) {
+            [[UIApplication sharedApplication] openURL:absoluteUrl];
+        } else {
+            // handle any custom schemes to plugins
```

Comment:
Is this something new from the plugin or is this something you have added?

Reply:
For future reference, this came from a separate plugin which opens the system browser directly. It was in our non wk-webview fork of the IAB